samples/dashboards/demo/accounting/demo.js

```diff
@@ -553,103 +542,3 @@ board.then(res => {
         ) / (revTarget - costTarget) * 100)
     });
 });
-
-/**
- * Popup
- */
-
-const popup = document.getElementById('popup');
-
-popup.addEventListener('click', e => {
-    if (e.target === popup || e.target.className === 'close') {
-        togglePopup(false);
-    }
-});
-
-document.addEventListener('keydown', e => {
-    if (e.key === 'Escape' && popup.style.display !== 'none') {
-        popup.style.display = 'none';
-    }
-});
-
-async function togglePopup(open) {
```

Comment:
What is the reason of removing popup from demo?

Reply:
It didn't work for some time and the data presented there wasn't important/easily readable.